ui/web/src/main/java/com/ttbmp/cinehub/ui/web/buyticket/BuyTicketPresenterWeb.java

```diff
@@ -1,149 +1,170 @@
 package com.ttbmp.cinehub.ui.web.buyticket;
 
+import com.ttbmp.cinehub.app.dto.ProjectionDto;
+import com.ttbmp.cinehub.app.dto.TicketDto;
 import com.ttbmp.cinehub.app.service.payment.PaymentServiceException;
 import com.ttbmp.cinehub.app.usecase.buyticket.BuyTicketPresenter;
 import com.ttbmp.cinehub.app.usecase.buyticket.request.*;
 import com.ttbmp.cinehub.app.usecase.buyticket.response.*;
+import org.springframework.ui.Model;
+
+import java.util.ArrayList;
+import java.util.List;
 
 /**
  * @author Palmieri Ivan
  */
 public class BuyTicketPresenterWeb implements BuyTicketPresenter {
 
+    private final Model model;
+    private String paymentError = "paymentError";
+    private String movieError = "movieError";
```

Comment:
This strings can be declared as static final, you should also find better names (i.e. [A-Z0-9_]*_ERROR_MESSAGE or something similar)